breakable_toy/Dockerfile-windshaft

```diff
@@ -0,0 +1,34 @@
+FROM node:0.10-slim
+
+MAINTAINER Azavea <systems@azavea.com>
+
+ENV WINDSHAFT_VERSION 0.36.0
+
+RUN mkdir -p /opt/windshaft
+
+WORKDIR /opt/windshaft
+
+RUN set -ex \
+  && buildDeps=' \
+    build-essential \
+    git-core \
+    libcairo2-dev \
+    libpango1.0-dev \
+    libjpeg62-turbo-dev \
+    libgif-dev \
+    libpq-dev \
+    ' \
+  && deps=' \
+    libcairo2 \
+    libgif4 \
+    libjpeg62-turbo \
+    libpixman-1-0 \
+    ' \
+  && apt-get update && apt-get install -y ${buildDeps} ${deps} --no-install-recommends \
+  && rm -rf /var/lib/apt/lists/* \
+  && npm install --unsafe-perm windshaft@${WINDSHAFT_VERSION} \
+  && apt-get purge -y --auto-remove ${buildDeps}
+
+EXPOSE 5000
+
+ENTRYPOINT ["node"]
```

Comment:
Because the [azavea/docker-windshaft](https://github.com/azavea/docker-windshaft) image is publicly available, we don't actually need the Dockerfile.  Everything this file is doing has already been done to make the published image and we can just start from that.  That's what the `image: ` line in the `docker-compose.yml` accomplishes.  (Actually, the presence of a `build` section affects the meaning of `image`. Without `build`, it means "use this existing image". With `build`, `image` changes meaning to "tag the built image with this name".)

If we did want a "similar but different" image, like if we needed to install some packages that aren't in the published image, copy/paste/modifying the Dockerfile would be one option, but in many cases it works to build on the published image.  I.e. make a Dockerfile with `FROM quay.io/azavea/windshaft:0.36.0` at the top then only specify the stuff we need in addition.  That's what the Windshaft container is doing--building on the [node:0.10-slim container](https://github.com/docker-library/node/blob/master/0.10/slim/Dockerfile).